README.md.template

```diff
@@ -351,7 +351,24 @@ $ solo node start
 
 # output is similar to example-1 
 ```
+## For Developer Working on Hedera Repo
 
+To set customized `settings.txt` file, edit the file
+`./resources/templates/settings.txt` before `node setup` command.
```

Comment:
For solo users, they need to edit the file in the solo cache directory: `~/.solo/cache/templates`
So correct path would be : `~/.solo/cache/templates/settings.txt` here.

Reply:
Readme template updated